src/copy_tree.rs

```diff
@@ -38,38 +40,71 @@ pub fn copy_tree<ST: ReadTree, DT: WriteTree>(
         // deleted or changed while this is running.
         ui::set_bytes_total(source.size()?.file_bytes);
     }
+
+    // Target selected for copy (subpaths from here and included)
+    let target = &options.restore_only;
```

Comment:
There's a bit of tension about whether this is a restore-specific option or generic to copy, as it's in generic code. Maybe call it `only_subtree`. 

I have been wondering about splitting the backup and restore paths, but for now better to keep it aligned.